dyno-jedis/src/main/java/com/netflix/dyno/jedis/JedisConnectionFactory.java

```diff
@@ -81,7 +81,7 @@ public JedisConnection(HostConnectionPool<Jedis> hostPool) {
 						hostPool.getSocketTimeout());
 			}
 			else {
-				jedisClient = new Jedis(host.getHostAddress(), host.getPort(), hostPool.getConnectionTimeout(),
+				jedisClient = new Jedis(host.getHostAddress(), host.getSecurePort(), hostPool.getConnectionTimeout(),
```

Comment:
Do we support secure port handling on Dynomite? Will this be half-baked if the server does not support it yet?

Reply:
what do you mean by half-baked? It depends on how cluster_describe says.